### PR TITLE
Add ConfigurationClasses

### PR DESCRIPTION
**Pull request by FedorivRostik**
Added ServicesConfiguration, MiddlewareConfiguration,RepositoriesConfiguration

### CODE REVIEW

**Review by Strafe153 — CHANGES_REQUESTED** [review]
Name some methods in the plural form

File: DataAccess/Repositories/ApplicationRepositoriesConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace DataAccess.Repositories
+{
+    public static class ApplicationRepositoriesConfiguration
+    {
+        public static void AddApplicationRepositorie(this IServiceCollection services)
```
Comment (Strafe153) [inline DataAccess/Repositories/ApplicationRepositoriesConfiguration.cs:7]:
```suggestion
        public static void AddApplicationRepositories(this IServiceCollection services)
```

File: Application/Services/ApplicationServicesConfiguration.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace Application.Services
+{
+    public static class ApplicationServicesConfiguration
+    {
+        public static void AddApplicationService(this IServiceCollection services)
```
Comment (Strafe153) [inline Application/Services/ApplicationServicesConfiguration.cs:7]:
```suggestion
        public static void AddApplicationServices(this IServiceCollection services)
```

File: Host/Program.cs [hunk 3]
```diff
@@ -1,13 +1,18 @@
+using Application.Services;
+using Host.Middleware;
+
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
+builder.Services.AddApplicationService();
```
Comment (Strafe153) [inline Host/Program.cs:7]:
```suggestion
builder.Services.AddApplicationServices();
```

**Review by YuraRov — CHANGES_REQUESTED** [review]
Please rename method "ApplicationServicesConfiguration.AddApplicationService" method to "ApplicationServicesConfiguration.AddApplicationServices" (plural naming) and the same thing for "ApplicationRepositoriesConfiguration" (Repositories).

**Review by Strafe153 — CHANGES_REQUESTED** [review]
Name some methods in the plural form

**Review by Strafe153 — APPROVED** [review]

File: Host/Middleware/ApplicationMiddlewareConfiguration.cs [hunk 4]
```diff
@@ -0,0 +1,12 @@
+﻿
+
+namespace Host.Middleware
+{
+    public static class ApplicationMiddlewareConfiguration
+    {
+        public static void AddApplicationMiddlewares(this IApplicationBuilder app)
```
Comment (Strafe153) [inline Host/Middleware/ApplicationMiddlewareConfiguration.cs:7]:
```suggestion
        public static void AddApplicationMiddleware(this IApplicationBuilder app)
```

**Review by Strafe153 — CHANGES_REQUESTED** [review]
Chagne middlewares to middleware

File: Host/Program.cs [hunk 5]
```diff
@@ -1,13 +1,18 @@
...
 // Add services to the container.
+builder.Services.AddApplicationServices();
 // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 
 var app = builder.Build();
 
 // Configure the HTTP request pipeline.
+app.AddApplicationMiddlewares();
```
Comment (YuraRov) [inline Host/Program.cs:15]:
```suggestion
app.AddApplicationMiddleware();
```

**Review by YuraRov — CHANGES_REQUESTED** [review]
The last thing (I promise): AddApplicationMiddleware, not AddApplicationMiddlewares

**Review by Strafe153 — APPROVED** [review]

**Review by YuraRov — APPROVED** [review]

**Pull request merged**